doc/architecture/modules/01_messages.md

```diff
@@ -0,0 +1,82 @@
+# Messages
+
+![Systemübersicht](../diagrams/modules/messages-overview.svg)
+
+_Systemübersicht._
+
+Mit Hitobito können Nachrichten (Briefe, SMS, Mails, usw...) an verschiedene Empfänger gesendet werden.
+
+## Abo
+![Systemübersicht](../diagrams/modules/messages-abo.svg)
+
+_Klassendiagramm des Abos._
+
+Damit Nachrichten Empfängern gesendet und zugeordnet werden können, verwendet man in Hitobito sogenannte Subscriptions, also Abonnements. Mit diesen wird sichergestellt, dass Personen oder Gruppen Abos haben können.
+
+
+### `Person` Model
+Das Model definiert alle Personen/Unternehmen welche in Hitobito vorhanden sind. 
```

Comment:
Person ist das zentrale Model in Hitobito für Personen und Firmen. Auf der Person sind auch die für das Message Modul relevanten Kontaktdaten wie E-Mail, Telefonnummer oder Postadresse gespeichert.